taraxa/state/dpos/precompiled/dpos_contract.go

```diff
@@ -819,25 +826,6 @@ func (self *Contract) registerValidator(ctx vm.CallFrame, block types.BlockNum,
 		return ErrMaxDescriptionLengthExceeded
 	}
 
-	// Make sure the public key is a valid one
-	pubKey, err := crypto.Ecrecover(args.Validator.Hash().Bytes(), args.Proof)
-	// the first byte of pubkey is bitcoin heritage
-	if err != nil {
-		return err
-	}
-
-	if common.BytesToAddress(keccak256.Hash(pubKey[1:])[12:]) != args.Validator {
-		return ErrWrongProof
-	}
-
-	if self.validators.ValidatorExists(&args.Validator) {
```

Comment:
this check should have probably stayed here. What is the reasoning behind removing it ?